packages/ckeditor5-dev-env/lib/release-tools/tasks/update-package-versions.js

```diff
@@ -0,0 +1,157 @@
+
+	options.pathsToUpdate.forEach( ( value, index, array ) => {
+		array[ index ].path = value.path.split( '\u005C' ).join( '\u002F' );
+	} );
+
+	console.log( '\n📍 ' + chalk.blue( 'Updating CKEditor 5 dependencies...\n' ) );
+
+	for ( const pathToUpdate of options.pathsToUpdate ) {
+		console.log( `Looking for package.json files in '${ pathToUpdate.path }'...` );
+
+		const result = updateDirectory( pathToUpdate.path, options.cwd, options.dryRun );
+
+		totalResult.found += result.found;
+		totalResult.updated += result.updated;
+
+		if ( result.differences ) {
+			totalResult.differences = [ ...totalResult.differences, ...result.differences ];
+		}
+
+		if ( !result.found ) {
+			console.log( 'No files were found.\n' );
+		} else if ( result.found && !result.updated ) {
+			console.log( `${ chalk.bold( result.found ) } files were found, but none needed to be updated.\n` );
+		} else {
+			console.log( `Out of ${ chalk.bold( result.found ) } files found, ${ chalk.bold( result.updated ) } were updated.\n` );
+
+			if ( pathToUpdate.commit ) {
+				totalResult.toCommit += result.updated;
+				pathsToCommit.push( pathToUpdate.path );
+			}
+		}
+	}
+
+	if ( options.dryRun ) {
+		console.log( chalk.yellow( 'DRY RUN mode - displaying changes instead of committing.' ) );
+
+		// TODO: Changes will be displayed here
+		console.log( totalResult );
+	} else if ( pathsToCommit.length ) {
+		console.log( '\n📍 ' + chalk.blue( 'Committing the changes...\n' ) );
+
+		for ( const path of pathsToCommit ) {
+			const execOptions = {
+				stdio: 'inherit',
+				cwd: path
+			};
+
+			console.log( `${ chalk.green( '+' ) } ${ path }` );
+
+			execSync( `git add ${ path }`, execOptions );
+			execSync( 'git commit -m "Internal: Updated all CKEditor 5 dependencies ' +
+				'in `packages/*` to the latest version. [skip ci]"', execOptions );
+		}
+
+		console.log( '\n📍 ' + chalk.green( `Successfully committed ${ totalResult.toCommit } files!\n` ) );
+	}
+
+	if ( totalResult.updated ) {
+		console.log( '\n📍 ' + chalk.green( `Updated total of ${ totalResult.updated } files!\n` ) );
+	} else {
+		console.log( '\n📍 ' + chalk.green( 'No files needed an update.\n' ) );
+	}
+};
+
+/**
+ * Updates `@ckeditor/ckeditor5-*` and `ckeditor5` dependencies in the specified directory to the latest version.
+ * Latest version is taken from the `version` property from the root of the `package.json` file, since that value
+ * should already have been bumped at this point of release process.
+ *
+ * @param {String} pathToUpdate directory containing files to update
+ * @param {String} cwd The root directory of the repository in which
+ * the script is being used.
+ * @returns {Object} Object containing numerical values counting files found and updated, as well as array of all changes made.
+ */
+function updateDirectory( pathToUpdate, cwd, dryRun ) {
+	const globPattern = pathToUpdate + '/*/package.json';
+	const packageJsonArray = glob.sync( globPattern, { cwd } );
```

Comment:
The `cwd` here is probably not needed and could be anything, as the `globPattern` is an absolute path.